src/uerouting.yaml

```diff
@@ -0,0 +1,3 @@
+info:
+  description: Routing information for UE
+  version: 1.0.0
```

Comment:
Is this needed? does it serve any functionality?

Reply:
The startup command needs the config file. I've had a populated version of it but it seems this gets populated by the webui/simapp and I've received a comment from Guilluame to remove it. His POC of charmed 5g has it.

Reply:
would the command fail if this file is not there? and wouldn't it be created when configured by `simapp`?

Reply:
Yes, it does.
```bash
2023-05-19T09:38:06Z [ERRO][SMF][CFG] open config/uerouting.yaml: no such file or directory
2023-05-19T09:38:06Z [ERRO][SMF][App] SMF Run error: Failed to initialize !!
```